core/probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/IntelliJApi.scala

```diff
@@ -81,13 +80,13 @@ trait IntelliJApi {
       result.asInstanceOf[B]
     }
 
-    def field[B: ClassTag](name: String): B = {
+    def field[B](name: String): B = {
       val field = getField(obj.getClass, "my" + name.capitalize)
       using(field)(_.get(obj).asInstanceOf[B])
     }
 
     private def using[B <: AccessibleObject, C](accessible: B)(f: B => C): C = {
-      val isAccessible = accessible.isAccessible
+      val isAccessible = accessible.canAccess(null) // isAccessible
```

Comment:
Whoops deprecated?

Pls clarify in the comment what you mean by `isAccessible` (it won't be clear soon)

Reply:
just forgot to remove the `// isAccessible` - will do that in next commit. `isAccessible` is deprecated. Moreover, I will have to double check if `canAccess(null)` should be used - or `canAccess(obj: Object)` with some real object... I am not sure yet after quick read of the source code of this method.

Reply:
Looks like we would have to stick to the `isAccessible` for now. Proper replacement with `canAccess` is not that obvious. Issue https://github.com/VirtusLab/ide-probe/issues/244 created.